node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -2440,26 +2437,37 @@ mod tests {
         let src_root = make_node_record(1234, true);
         let dest_root = make_node_record(2345, true);
         let mut src_db = db_from_node(&src_root);
-        let node_a_fr = make_node_record_cc(5678, true, "FR");
-        let node_b_us = make_node_record_cc(4567, true, "US");
+        let node_a = make_node_record(5678, true);
+        let country_zero = node_a.inner.country_code_opt.clone().unwrap();
```

Comment:
I think there's a way you can iterate on `COUNTRIES` to get codes instead of IP addresses; then you should be able to iterate until you find a country code that's not `country_zero` or `ZZ` or `NOEX`.